src/components/video.tsx

```diff
@@ -26,7 +26,7 @@ export function VimeoEmbed({id, className}: Video) {
   return (
     <ResponsiveEmbed className={className}>
       <StyledVimeoIframe
-        src={`https://player.vimeo.com/video/${id}?dnt=true`}
+        src={`https://player.vimeo.com/video/${id}&dnt=true`}
```

Comment:
if other `<VimeoEmbed>` do not include `?` after the id this wont work I assume, did you check for other occurrences?